app/views/homepage/_popular_links.html.erb

```diff
@@ -41,7 +41,7 @@
                     'index_section': locals[:index_section],
                     'index_link': index + 1,
                     'index_section_count': locals[:index_section_count],
-                    'index_total': t("homepage.index.popular_links").length,
+                    'index_total': t("homepage.index.popular_links_a_z").length,
```

Comment:
We will iterate the links for each variant based on the analytics I assume, and I've not read anything to suggest we are bound to there being 6. So I think it would be nicer to not have this "hardcoded" so any iterations would just be changes to the yaml file. You've already got the `popular_links` variable defined at the top of the file, so you could use that here right? `popular_links.length`